CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# dbt_zendesk v0.9.0
```

Comment:
I feel we should also add the `_zendesk_source` schema change notes here as well. Just so people that notice the change can see all relevant notes here.